nullaway/src/main/java/com/uber/nullaway/handlers/Handler.java

```diff
@@ -402,6 +402,19 @@ Integer castToNonNullArgumentPositionsForMethod(
       List<? extends ExpressionTree> actualParams,
       @Nullable Integer previousArgumentPosition);
 
+  /**
+   * Called to get the library models which have Nullable upper bound for their Generic Type
+   * Arguments.
+   *
+   * @param className A String containing the name of the class for which we want to check Nullable
+   *     upper bound.
+   * @param index Indicates the generic type argument for which we want to check Nullable upper
+   *     bound.
+   * @return boolean value of regardless of whether the upper bound for that argument has a Nullable
+   *     upper bound.
+   */
+  boolean onOverrideTypeParameterUpperBound(String className, int index);
+
```

Comment:
Didn't I suggest some changes in these docs before?  Did those changes get lost?

Reply:
I added that in a javadoc for the CompositeHandler since it seemed specific to the implementation inside CompositeHandler